### PR TITLE
PW-6010 defactor checkout services controller file

### PR DESCRIPTION
**Pull request by zaiddreakh**
## Summary

This PR reverts the controller file `CheckoutServices.js` back to its original form (core code)
The Adyen cartridge blocks of code that are added to the file are wrapped with comments that indicate start and end for each block

## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/CheckoutServices.js [hunk 1]
```diff
@@ -1,18 +1,230 @@
...
- * when the selected payment processor is Adyen.
- */
-server.prepend(
-  'PlaceOrder',
-  server.middleware.https,
-  checkoutServices.placeOrder,
-);
-
-module.exports = server.exports();
+const adyenHelpers = require('*/cartridge/scripts/checkout/adyenHelpers');
```
Comment (wboereboom) [inline src/cartridges/int_adyen_SFRA/cartridge/controllers/CheckoutServices.js:6]: add `/* ### Custom Adyen cartridge ### */` tags around imports

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/checkout_services/adyenCheckoutServices.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  res.json({
+    error: false,
+    adyenAction: handlePaymentResult.action,
+    orderID: order.orderNo,
+    orderToken: order.orderToken,
+  });
+  emit('route:Complete');
+}
+
+function isAdyen(currentBasket, next) {
```
Comment (wboereboom) [inline src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/checkout_services/adyenCheckoutServices.js:14]: this function return type is inconsistent, it a boolean, or `next`. Unless next is a boolean, we could consider removing it from this function to make it more consistent and do
```
if(isAdyen) {
  return next();
}
```
in CheckoutServices.js

**Review by wboereboom — COMMENTED** [review]
2 small comments. looks good otherwise! 👍

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/CheckoutServices.js [hunk 3]
```diff
@@ -33,7 +35,9 @@ server.prepend('PlaceOrder', server.middleware.https, function (req, res, next)
   }
 
   /* ### Custom Adyen cartridge ### */
-  isAdyen(currentBasket, next);
+  if(isNotAdyen(currentBasket)) {
```
Comment (wboereboom) [inline src/cartridges/int_adyen_SFRA/cartridge/controllers/CheckoutServices.js:38]: I like this. very human readable 👌 . great work.

**Review by wboereboom — DISMISSED** [review]

**Review by wboereboom — APPROVED** [review]

**Review by michaelpaul — APPROVED** [review]

**Pull request merged**